Utils/hdinsight-node-common/Test/java/com/microsoft/azure/hdinsight/spark/common/SparkBatchRemoteDebugJobScenario.java

```diff
@@ -280,4 +273,15 @@ public void gettingSparkJobDriverLogURLObservableShouldBeEmpty() throws Throwabl
     public void mockSparkJobUriIsValidOrNot(String uriToCheck, String validOrNot) throws Throwable {
         doReturn(validOrNot.equals("valid")).when(debugJobMock).isUriValid(URI.create(uriToCheck));
     }
+
+    @And("^submit Spark job$")
```

Comment:
>@And("^submit Spark job$") [](start = 4, length = 26)

This test is for Spark debug job?

Reply:
This test feature, Spark Job Spark Debug Job cases are mixed, so the naming here is for both.